lib/initCookieSession.js

```diff
@@ -1,10 +1,28 @@
 const cookieSession = require('cookie-session');
 const config = require('./config');
+//Cookie max age variant for environments 30mins prod 2 elseware
+//TODO make config var?
+//const maxAge = (config.NODE_ENV === 'production')? 30 : 2;
 
 const sessionDetails = {
-	name: 'FTkmt',
+	name: 'FTKat',
```

Comment:
You renamed this to `FTKat` here but left it as `FTkmt` here `https://github.com/Financial-Times/kmt-utilities/blob/f42a92e73d880c712cfd6a6120bd4b07dd81442b/lib/helpers/cookieHandler.js#L15`